test/e2e/atlas/clusters_m0_test.go

```diff
@@ -39,9 +39,6 @@ func TestClustersM0Flags(t *testing.T) {
 	clusterName, err := RandClusterName()
 	req.NoError(err)
 
-	mongoDBMajorVersion, err := MongoDBMajorVersion()
```

Comment:
right now the clusters team is looking into splitting the update of default mdb version and the version for the shared tier which historically were changed at the same time, can you create a ticket to investigate next steps here to avoid having to update this test every year, this also kind of affects the clusters command which the help may say an M0 is created with 7.0 but it'll show as 8.0

Reply:
do you have any references to `right now the clusters team is looking into splitting the update of default mdb version and the version for the shared tier` ?